src/Microsoft.Extensions.DiagnosticAdapter/DiagnosticSourceAdapter.cs

```diff
@@ -79,7 +125,7 @@ public void Write(string diagnosticName, object parameters)
                 return;
             }
 
-            if (_listener.IsEnabled != null && !_listener.IsEnabled(diagnosticName))
+            if (_listener.IsEnabled != null && !_listener.IsEnabled(diagnosticName, null, null))
```

Comment:
I am not entirely sure. I think it can be any context we want to associate with the events. https://github.com/dotnet/corefx/issues/15984#issuecomment-278692599